calculator.js

```diff
@@ -3,3 +3,51 @@ const exampleAdditionInput = {
   num2: 5,
   operation: 'add',
 }
+
+const calculate = function (inputHash) {
+  leftOperator = inputHash.num1;
+  rightOperator = inputHash.num2;
+  operand = inputHash.operation;
+
+  if (typeof(leftOperator) !== 'number' || typeof(rightOperator) !== 'number') {
+    return('Sorry, your operators are not valid!');
+  }
+
+  switch (operand) {
+    case 'add':
+      return (leftOperator + rightOperator);
+    case '+':
+      return (leftOperator + rightOperator);
+    case 'subtract':
+      return (leftOperator - rightOperator);
+    case '-':
+      return (leftOperator - rightOperator);
+    case 'multiply':
+      return (leftOperator * rightOperator);
+    case '*':
+      return (leftOperator * rightOperator);
+    case 'divide':
+      return (leftOperator/rightOperator);
+    case '/':
+      return (leftOperator/rightOperator);
+    default:
+      return ('That wasn\'t a valid operand!');
+  }
```

Comment:
Since you mentioned wanting to DRY this up, you can clean this up using what we call "case fall through".  Basically it means if you stack two cases on top of each other without a `return` or a `break` they will both run the same code:

```suggestion
  switch (operand) {
    case 'add':
    case '+':
      return (leftOperator + rightOperator);
    case 'subtract':
    case '-':
      return (leftOperator - rightOperator);
    case 'multiply':
    case '*':
      return (leftOperator * rightOperator);
    case 'divide':
    case '/':
      return (leftOperator/rightOperator);
    default:
      return ('That wasn\'t a valid operand!');
  }
```